src/ChannelAllocationChromosome.java

```diff
@@ -61,4 +61,7 @@ public Float getGene(int i) {
         return genes.get(i);
     }
 
+    public MetaData getMetaData() {
+        return metaData;
+    }
```

Comment:
```suggestion
```
MetaData will be send in the constructor of the ChannelAllocationChromosomeMutation